odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/reactive/ReactiveEnumValueRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.opendatadiscovery.oddplatform.repository.reactive;
+
+import java.util.List;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.EnumValuePojo;
+import reactor.core.publisher.Flux;
+
+public interface ReactiveEnumValueRepository extends ReactiveCRUDRepository<EnumValuePojo> {
+    Flux<EnumValuePojo> getEnumValuesByFieldId(final long datasetFieldId);
+
+    Flux<EnumValuePojo> softDeleteOutdatedEnumValues(final long datasetFieldId, final List<Long> idsToKeep);
```

Comment:
let's rename this to `softDeleteOutdatedEnumValuesExcept` . This will reduce confusion about `idsToKeep` parameter.